llvm/utils/git/github-automation.py

```diff
@@ -567,9 +569,13 @@ def create_pull_request(self, owner: str, repo_name: str, branch: str) -> bool:
             print("PR already exists...")
             return True
         try:
+            commit_message = repo.get_commit(commits[-1]).commit.message
+            message_lines = commit_message.splitlines()
+            title = "{}: {}".format(release_branch_for_issue, message_lines[0])
+            body = "Backport {}".format(" ".join(commits))
```

Comment:
I'd suggest to also add `requested by @{}` or so here. Currently, I find it hard to keep track of backports that I have requested, because I'm not subscribed to the PRs and don't get notifications about activity.